src/main/java/frc/robot/Robot.java

```diff
@@ -118,5 +121,17 @@ public void testPeriodic() {
 
   private void manualControl() {
     drive.arcadeDrive(controls.getLeftY(DEADZONE), controls.getRightX(DEADZONE) *0.75);
+    //System.out.println("verticalSpeed" + controls.getLeftY(DEADZONE));
+   // System.out.println("horizontalSpeed" + controls.getRightX(DEADZONE));
+   // System.out.println("LeftEncoder" + drive.leftEncoder.getDistance());
+   // System.out.println("RightEncoder"+ drive.rightEncoder.getDistance());
+
+    if(controls.getBButton())
+    {
+      drive.driveStraight(36, .75);
```

Comment:
Let's make these numbers configurable via the dashboard and move this code to autonomous.